.travis.yml

```diff
@@ -69,7 +69,7 @@ matrix:
           - git clone https://github.com/canonical/server-test-scripts.git
       script:
           - ./tools/run_aws_travis_integration_tests.sh
-    - if: env(UACLIENT_BEHAVE_AWS_SECRET_ACCESS_KEY) AND env(UACLIENT_BEHAVE_CONTRACT_TOKEN)
+    - if: env(UACLIENT_BEHAVE_AWS_SECRET_ACCESS_KEY) AND env(UACLIENT_BEHAVE_CONTRACT_TOKEN) AND env(TRAVIS_PULL_REQUEST) = "false"
```

Comment:
it feels more explicit to say env(TRAVIS_EVENT_TYPE) = cron because that's really the only place we want to run the breadth of these cloud tests